### PR TITLE
add plugin doc generator

### PR DESCRIPTION
**Pull request by EvanLjp**
add add plugin doc generator

### CODE REVIEW

**Comment by wu-sheng** [comment]
CI fails, please fix.

**Comment by EvanLjp** [comment]
> CI fails, please fix.

fixing

**Review by wu-sheng — APPROVED** [review]

**Review by kezhenxu94 — APPROVED** [review]

File: internal/satellite/tools/generate_plugin_doc.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	"strings"
+
+	"github.com/apache/skywalking-satellite/internal/pkg/log"
+	"github.com/apache/skywalking-satellite/internal/pkg/plugin"
+	"github.com/apache/skywalking-satellite/plugins"
+)
+
+const (
+	docDir      = "docs"
+	docPath     = docDir + "/plugin-description.md"
```
Comment (nic-chen) [inline internal/satellite/tools/generate_plugin_doc.go:35]: we could use `filepath.Join` instead of concatenating them manually.

Comment (moonming) [reply internal/satellite/tools/generate_plugin_doc.go:35]: welcome PR to fix it.

Comment (membphis) [reply internal/satellite/tools/generate_plugin_doc.go:35]: I think you can submit a PR directly ^_^ @nic-chen

Comment (nic-chen) [reply internal/satellite/tools/generate_plugin_doc.go:35]: sure, thanks for remind.

Comment (kezhenxu94) [reply internal/satellite/tools/generate_plugin_doc.go:35]: Using a function in `const` is not allowed, unless you want to change all `const` to `var`

Comment (nic-chen) [reply internal/satellite/tools/generate_plugin_doc.go:35]: @kezhenxu94 
you're right. There is little benefit from such modification.
I submitted another pr, welcome to review, thanks.

https://github.com/apache/skywalking-satellite/pull/15

**Pull request merged**